docs/camino-node/auto-upgrade.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 6
+title: Auto Upgrade Node
+description: Automatically Upgrade Camino Node
+---
+
+# Automatically Upgrade Camino Node
+
+This guide provides instructions on how to automatically upgrade camino-node using a bash script and crontab.
+
+## Automatic Install Script
+
+If you have previously installed camino-node using the autoinstall script, you can now configure an automatic
+upgrade with the following steps:
+
+### Download Node Upgrader
+
+First, download the node upgrader script. You can place this script anywhere on the node's operating system.
+For the sake of this guide, we will assume you put it under the `/root` directory.
+
+```
+wget -nd -m https://raw.githubusercontent.com/chain4travel/camino-docs/c4t/scripts/camino-node-upgrader.sh;\
+chmod 755 camino-node-upgrader.sh
+```
+
+**Usage:**
+
+The script accepts a single option `-u` followed by a username argument. Provide the username used to install camino-node.
+
+```
+# ./camino-node-upgrader.sh -h
+Usage: camino-node-upgrader.sh -u username
+```
+
+:::note DESINGED TO BE RUN AS ROOT
+
+This script is intended to be executed with root user privileges.
+
+:::
+
+### Edit Crontab File
+
+Edit the crontab file and add the following line. Choose the desired time for the cron service to run the script.
```

Comment:
maybe tell them where the file is located. /etc/crontab

Reply:
Added